xml/ha_storage_protection.xml

```diff
@@ -809,17 +809,50 @@ Timeout (msgwait)  : 10
      </callout>
    </calloutlist>
   </step>
-  <step>
-   <para>For a two-node cluster, decide if you want predictable or random delays. For
-   other cluster setups you do not need to set this parameter.</para>
+  <step xml:id="st-ha-storage-protect-fencing-static-random">
+   <para>
+    For a two-node cluster, in case of split-brain, there will be fencing issued from
+    each node to the other as expected. To prevent both nodes from practically
+    being reset at the same time, it is recommended to apply the following fencing
+    delays to help one of the nodes, or even the preferred node, win the fencing match.
+    For clusters with more than two nodes, you do not need to apply these delays.
+   </para>
    <variablelist>
+    <varlistentry>
+     <term>Priority fencing delay</term>
+     <listitem>
+       <para>
+        By configuring the <literal>priority-fencing-delay</literal> cluster property
+        with a delay value (by default disabled), if the other node is lost and it has
+        the higher total resource priority, the fencing targeting it will be delayed for
+        the specified amount of time. So that the more important node wins fencing
+        match in case of split-brain.
+      </para>
+      <para>
+        Resources that matter can be configured with priority meta attribute. On
+        calculation, the priority values of the resources or instances that are running
+        on a node are summed up to be accounted. A promoted resource instance takes the
```

Comment:
It seems it'd be even better to go with "on `each` node are summed up to be accounted"?

Reply:
Yes, that's better. :+1: Corrected that.